app/policies/agent/agent_policy.rb

```diff
@@ -11,6 +11,7 @@ def current_agent_or_admin_in_record_organisation?
   alias show? current_agent_or_admin_in_record_organisation?
   alias new? current_agent_or_admin_in_record_organisation?
   alias create? current_agent_or_admin_in_record_organisation?
+  alias index? current_agent_or_admin_in_record_organisation?
```

Comment:
ca me perturbe pas mal semantiquement ca veut dire : un agent a le droit de lister les autres agents des orgas ou il est admin. as-tu vraiment besoin de cette nouvelle permission

Reply:
:thinking: il est possible que ce soit un résidu de quelque chose. Mais je crois que c'est lié au `authorize_admin(agent)` qui est fait dans la méthode `index` des controlleurs `app/controller/admin/agents/(plage_ouvertures|absences|rdvs)_controller.rb`. Je me trompe peut-être. Je vais essayer de l'enlever pour voir ce qu'il se passe :)

Reply:
Je viens de vérifier, j'en ai besoin à cause du `authorize_admin(agent)` de la méthode `index`.

Reply:
ah oui et bien en effet le probleme c'est exactement ça : on ne devrait pas faire de `authorize(current_agent)` dans `rdvs#index`, ni `absences#index` etc. 

cf https://github.com/varvet/pundit#ensuring-policies-and-scopes-are-used ils recommandent generalement de ne pas forcer a utiliser authorize sur la route index. 
sur cette route seul `policy_scope` suffit

Reply:
Ok, merci. Je supprime les `authorize` et cette ligne.